Updating ember & ember-cli to 2.15

mansona opened the pull request:
I tried to make this as minimal as possible, essentially it should be a simple enough merge. It passes the tests and should not have changed anything. 

One of the added benefits is that it no longer needs bower 🎉

mansona committed:
updating ember-cli version

mansona commented:
Howdy folks, I know it hasn't been too long since I submitted this PR but I'm wondering if anyone has had a chance to take a look at it? 

I'm not trying to rush you, I'm just going to take a crack at getting ember-cli-rollbar **working properly** in a fastboot environment and having an up-to-date repo should make a big difference 👍 

I don't even need this to be merged before I start, I would just like a nod to say that it is likely to be merged (with whatever changes you need) so that I can branch my new work from this commit instead of current master

davewasmer commented:
@mansona thanks for this! Sorry for the delayed response here ... house renovations taking up too much time!

Happy to merge this, and I appreciate the effort. Only one changed I'd ask - mind dropping ember-suave? I'm not a big fan of the rules included in that set.

mansona committed:
removing ember-suave

mansona commented:
@davewasmer no worries on the delayed response, we all got stuff to do 😉 

I've removed ember-suave and also rebased as I saw you merged in a few other things 👍  let me know if you want me to change anything else

mansona commented:
@davewasmer I didn't quite understand why the tests were failing but it looks like ember-try uses yarn and yarn is having some issues with the dependencies. 

I have updated ember-try in the package.json so that the tests will pass. 

you can see more on the issue here: https://github.com/ember-cli/ember-try/pull/138

mansona committed:
updating ember-try to fix tests

mansona committed:
updating to 2.15

mansona commented:
I figured I would update it to 2.15 since that was released since my original PR 👍

mansona commented:
@davewasmer any movement on this? Let me know if you would like anything else changed

davewasmer commented:
Looks great! Thanks @mansona!
